workos/src/main/kotlin/com/workos/directorysync/DirectorySyncApi.kt

```diff
@@ -83,19 +112,40 @@ class DirectorySyncApi(private val workos: WorkOS) {
       if (user != null) set("user", user)
     }
 
+    /**
+     * @suppress
+     */
     companion object {
       @JvmStatic
-      fun builder(): Builder {
-        return Builder()
+      fun builder(): ListDirectoryGroupOptionsBuilder {
+        return ListDirectoryGroupOptionsBuilder()
       }
     }
 
-    class Builder : PaginationParams.Builder<ListDirectoryGroupOptions>(ListDirectoryGroupOptions()) {
+    /**
+     * Builder class for creating [ListDirectoryGroupOptions]
+     */
+    class ListDirectoryGroupOptionsBuilder : PaginationParams.Builder<ListDirectoryGroupOptions>(ListDirectoryGroupOptions()) {
+      /**
+       * The directory id to filter on.
+       */
       fun directory(value: String) = apply { this.params["directory"] = value }
+      /**
+       * The user id to filter on.
+       */
       fun user(value: String) = apply { this.params["user"] = value }
     }
   }
 
+  /**
+   * Parameters for [listDirectoryUsers]
+   *
+   * @param directory the id of the directory to list user for
```

Comment:
Been a while since I’ve done Java, but IIRC it’s idiomatic to treat the parameter description as a proper sentence:
```suggestion
   * @param directory The ID of the directory to list the user for.
```